spec/controllers/users_controller_spec.rb

```diff
@@ -70,7 +70,6 @@
       context 'when using "paged" paginator' do
         before(:all) do
           JSONAPI.configuration.default_paginator = :paged
-          UserResource.paginator :paged
```

Comment:
Previously it also needed to set the paginator in the resource level because of a JR's weird behavior. Good to know that it's no longer required.